url/urlsearchparams-constructor.html

```diff
@@ -18,15 +18,25 @@
 }, 'Basic URLSearchParams construction');
 
 test(function() {
-    assert_throws(new TypeError(), function () { URLSearchParams(); },
-                  'Calling \'URLSearchParams\' without \'new\' should throw.');
-    assert_throws(new TypeError(), function () { new URLSearchParams(DOMException.prototype); });
-    var params = new URLSearchParams('');
+    var params = new URLSearchParams()
+    assert_equals(params.toString(), "")
+}, "URLSearchParams constructor, no arguments")
+
+test(() => {
+    params = new URLSearchParams(DOMException.prototype);
+    assert_equals(params.toString(), "Error=")
```

Comment:
Can you clarify why this is the expected result here. My interpretation of the Web IDL spec [1] is that we should go to step 11.1 and treat the input as a sequence (not a String).

[1] https://heycam.github.io/webidl/#es-union